pages/api/export.js

```diff
@@ -0,0 +1,84 @@
+import React from "react";
+import { renderToString } from "react-dom/server";
+import pdf from "html-pdf";
+import {
+  ThemeProvider as MuiThemeProvider,
+  ServerStyleSheets as MuiServerStyleSheet,
+} from "@material-ui/core/styles";
+import {
+  ThemeProvider as StyledThemeProvider,
+  ServerStyleSheet as StyledServerStyleSheet,
+  StyleSheetManager,
+} from "styled-components";
+import CssBaseline from "@material-ui/core/CssBaseline";
+import { AppContainer } from "@licenserocks/kit";
+
+import { Icons } from "theme/icons";
+import { config } from "config";
+import { getLicenseInfo } from "utils/ethereum";
+import { CertificatePDF } from "components";
+
+function renderFullPage(html, muiStyleTags, scStyleTags) {
+  return `
+    <!DOCTYPE html>
+    <html>
+      <head>
+        <title>My page</title>
+        ${muiStyleTags}
+        ${scStyleTags}
+      </head>
+      <body>
+        <div id="root">${html}</div>
+      </body>
+    </html>
+  `;
+}
+
+export default async function user(req, res) {
+  const { id, network, contractAddr } = req.query;
+  const licenseInfo = await getLicenseInfo(id, contractAddr, network);
+  const { license } = licenseInfo;
+
+  const { theme } = config;
+
+  const muiSheets = new MuiServerStyleSheet();
+  const styledSheets = new StyledServerStyleSheet();
+  const markup = renderToString(
+    muiSheets.collect(
+      <StyleSheetManager sheet={styledSheets.instance}>
+        <MuiThemeProvider theme={theme}>
+          <StyledThemeProvider theme={theme}>
+            <CssBaseline />
+            <AppContainer appConfig={config} theme={theme} icons={Icons}>
```

Comment:
`AppContainer` component already contains `MuiThemeProvider`, `StyledThemeProvider`, `CssBaseline`. Please remove them.

Reply:
@magiiixam1997 If I remove these props, I see the error message attached in the following image. Maybe the problem is that I'm using this component in an API endpoint.
![Screen Shot 1399-09-06 at 13 54 53](https://user-images.githubusercontent.com/72013348/100339565-436f7800-2fef-11eb-9648-1041f1579541.png)